server/app/boot.rb

```diff
@@ -30,3 +30,5 @@
 Dir[__dir__ + '/jobs/**/*.rb'].each {|file| require file }
 
 Dir[__dir__ + '/services/**/*.rb'].each {|file| require file }
+
+Mongodb::Migrator.new.migrate
```

Comment:
Still not a correct place for this.. this problem is already fixed in the master so imho this PR should not try to re-implement it differently? :)

See #999

/cc @jnummelin @nevalla